test/known_issues/test-http2-compat-head-finished.js

```diff
@@ -0,0 +1,37 @@
+'use strict';
+
+const common = require('../common');
+if (!common.hasCrypto)
+  common.skip('missing crypto');
```

Comment:
@mcollina For future reference, what is the purpose of this? I just followed the other tests, not sure if or why it would be needed?

Reply:
@ronag The `http2` module requires encryption, so if Node.js was compiled without crypto, then `http2` tests need to be skipped.